### DISCUSSION TITLE
Generate My Own QR Codes in C++. Are Labs QR Codes Just Plain Text?

### DISCUSSION THREAD

**Discussion by foodfo** [gopro/labs#895] [Q&A] [answered]
Are the GoPro Labs QR codes just plain text encoded into a QR code?
I want to be able to generate my own QR codes in C++.  I need to pass in variables from my own software and output a QR code image to change the burn-in text on a gopro.

**Comment by dnewman-gpsw** [comment, marked as answer]
Just plain text, any QR Code generator will work. This example is just Javascript running input from an URL, so you don't enough need C++  https://gopro.github.io/labs/control/set/?cmd=!SM3D4H5!R&title=Motion%20Detection
